### ISSUE TITLE
multipass list error:   list failed: ssh connection failed: 'No route to host'

### ISSUE THREAD

**Issue by watcher00090** [canonical/multipass#2070]
**Describe the bug**
`multipass list`  sometimes fails if run while some VMs are in the process of stopping. 
```
james@james-ThinkStation-P330-Tiny:~$ multipass list
list failed: ssh connection failed: 'No route to host'
```

**To Reproduce**

1. `multipass stop my_vm` (where my_vm is an Ubuntu 20.04 LTS VM) 
2. Immediately afterwards, run:`multipass list`

**Expected behavior**
`multipass list` should run without error even while some VMs are in the process of stopping. Currently it takes about 15 seconds after shutting down a VM for `multipass list` to run successfully. 

**Logs**

```
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: process working dir '/snap/multipass/3531/qemu'
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: process program 'qemu-system-x86_64'
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: process arguments '--enable-kvm, -device, virtio-scsi-pci,id=scsi0, -drive, file=/var/snap/multipass/common/data/multipassd/vault/instances/mayastor-cluster-master/ubuntu-20.04-server-cloudimg-amd64.img,if=none,format=qcow2,discard=unmap,id=hda, -device, scsi-hd,drive=hda,bus=scsi0.0, -smp, 2, -m, 5120M, -device, virtio-net-pci,netdev=hostnet0,id=net0,mac=52:54:00:8b:db:7a, -netdev, tap,id=hostnet0,ifname=tap-7599c877141,script=no,downscript=no, -qmp, stdio, -cpu, host, -chardev, null,id=char0, -serial, chardev:char0, -nographic, -cdrom, /var/snap/multipass/common/data/multipassd/vault/instances/mayastor-cluster-master/cloud-init-config.iso'
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: [1849607] started: qemu-system-x86_64 -nographic -dump-vmstate /tmp/multipassd.ntQ944
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: qemu-system-x86_64: warning: TCG doesn't support requested feature: CPUID.01H:ECX.vmx [bit 5]
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[1849610]: Applying AppArmor policy: multipass.mayastor-cluster-master.qemu-system-x86_64
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: [1849610] started: v --enable-kvm -device virtio-scsi-pci,id=scsi0 -drive file=/var/snap/multipass/common/data/multipassd/vault/instances/mayastor-cluster-master/ubuntu-20.04-server-cloudimg-amd64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 5120M -device virtio-net-pci,netdev=hostnet0,id=net0,mac=52:54:00:8b:db:7a -netdev tap,id=hostnet0,ifname=tap-7599c877141,script=no,downscript=no -qmp stdio -cpu host -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/snap/multipass/common/data/multipassd/vault/instances/mayastor-cluster-master/cloud-init-config.iso
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: process started
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: Trying SSH on 10.239.179.135:22
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: QMP: {"QMP": {"version": {"qemu": {"micro": 1, "minor": 11, "major": 2}, "package": "(Debian 1:2.11+dfsg-1ubuntu7.36)"}, "capabilities": []}}
Apr 22 17:38:30 james-ThinkStation-P330-Tiny multipassd[944]: QMP: {"return": {}}
Apr 22 17:38:37 james-ThinkStation-P330-Tiny multipassd[944]: QMP: {"timestamp": {"seconds": 1619138317, "microseconds": 291231}, "event": "NIC_RX_FILTER_CHANGED", "data": {"name": "net0", "path": "/machine/peripheral/net0/virtio-backend"}}
Apr 22 17:38:37 james-ThinkStation-P330-Tiny dnsmasq-dhcp[1601]: DHCPDISCOVER(mpqemubr0) 52:54:00:8b:db:7a
Apr 22 17:38:37 james-ThinkStation-P330-Tiny dnsmasq-dhcp[1601]: DHCPOFFER(mpqemubr0) 10.239.179.135 52:54:00:8b:db:7a
Apr 22 17:38:37 james-ThinkStation-P330-Tiny dnsmasq-dhcp[1601]: DHCPREQUEST(mpqemubr0) 10.239.179.135 52:54:00:8b:db:7a
Apr 22 17:38:37 james-ThinkStation-P330-Tiny dnsmasq-dhcp[1601]: DHCPACK(mpqemubr0) 10.239.179.135 52:54:00:8b:db:7a mayastor-cluster-master
Apr 22 17:38:48 james-ThinkStation-P330-Tiny multipassd[944]: executing 'ip -brief -family inet address show scope global'
Apr 22 17:38:48 james-ThinkStation-P330-Tiny multipassd[944]: Executing 'ip -brief -family inet address show scope global'
Apr 22 17:38:49 james-ThinkStation-P330-Tiny multipassd[944]: /build/multipass/parts/multipass/src/src/ssh/ssh_process.cpp:117 read_stream(type = 0, timeout = -1):
Apr 22 17:38:49 james-ThinkStation-P330-Tiny multipassd[944]: /build/multipass/parts/multipass/src/src/ssh/ssh_process.cpp:135 read_stream(): num_bytes = 143
Apr 22 17:38:49 james-ThinkStation-P330-Tiny multipassd[944]: /build/multipass/parts/multipass/src/src/ssh/ssh_process.cpp:135 read_stream(): num_bytes = 0
Apr 22 17:38:49 james-ThinkStation-P330-Tiny multipassd[944]: QMP: {"timestamp": {"seconds": 1619138329, "microseconds": 566172}, "event": "RTC_CHANGE", "data": {"offset": 1}}
Apr 22 17:38:56 james-ThinkStation-P330-Tiny multipassd[944]: Executing 'wall The system is going down for poweroff now'
Apr 22 17:38:56 james-ThinkStation-P330-Tiny multipassd[944]: No mounts to stop for instance "mayastor-cluster-master"
Apr 22 17:38:56 james-ThinkStation-P330-Tiny multipassd[944]: QMP: {"return": {}}
                                                              {"timestamp": {"seconds": 1619138336, "microseconds": 796393}, "event": "POWERDOWN"}
Apr 22 17:40:33 james-ThinkStation-P330-Tiny multipassd[944]: QMP: {"timestamp": {"seconds": 1619138427, "microseconds": 706044}, "event": "SHUTDOWN", "data": {"guest": true}}
Apr 22 17:40:33 james-ThinkStation-P330-Tiny multipassd[944]: VM shut down
Apr 22 17:40:33 james-ThinkStation-P330-Tiny multipassd[944]: process finished with exit code 0
```

**Additional info**
 - OS: Ubuntu 20.04 LTS
- Multipass versions: multipass 1.6.2,  multipassd 1.6.2

**Additional context**
local.driver = qemu

**Comment by Saviq** [comment]
Hi @watcher00090, that's fixed in #1993, you can use the `edge` channel for now or wait for the next `stable` in a couple weeks.

**Comment by Saviq** [comment]
Duplicate of #1992
